.codeclimate.yml

```diff
@@ -1,4 +1,6 @@
 version: "2"         # required to adjust maintainability checks
+exclude_patterns:
+  - "**/*_test.go"
```

Comment:
Why? :smile:

Reply:
Too many "similar code", "complexity" in tests code, we should exclude them from CodeClimate. And I need to change pattern for storage/store/test.go !